src/modules/household/components/ManageHousehold.tsx

```diff
@@ -32,21 +32,22 @@ import {
   ClientSortOption,
   EnrollmentFieldsFragment,
   ExternalIdentifierType,
+  ProjectAllFieldsFragment,
   SearchClientsDocument,
   SearchClientsQuery,
   SearchClientsQueryVariables,
 } from '@/types/gqlTypes';
 
 interface Props {
   householdId?: string;
-  projectId: string;
+  project: Pick<ProjectAllFieldsFragment, 'id' | 'projectName' | 'access'>;
```

Comment:
Was this an intermediate change that didnt end up getting used?

Reply:
It's used in the `useAddToHouseholdColumns` hook args on line 69